ip_country/src/country_finder.rs

```diff
@@ -110,7 +71,7 @@ mod tests {
             CountryCodeFinder::new(ipv4_country_data(), ipv6_country_data());
     }
 
-    pub(crate) fn ipv4_country_data() -> (Vec<u64>, usize) {
+    pub fn ipv4_country_data() -> (Vec<u64>, usize) {
```

Comment:
Are you sure about the second returned value in the tuple? Is it or it isn't important that it corresponds to the count of the vector's items? (I'm aware it's test data but still)

Reply:
This is probably question for Dan. I did not created or reviewd this piece of code. And I assume, that nuber is right. But I can add test for this. It is really simple.